### DISCUSSION TITLE
Install Github desktop Powershell

### DISCUSSION THREAD

**Discussion by kearve** [community/community#109343] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello All,

I the past I was able to install github desktop using Powershell, however the same commands did not worked for me with the latest, have anyone experience the same? And was able to resolve and install using command line or Powershell for Windows.

Your help will be highly appreciated.

**Comment by steveward** [comment]
@kearve I'm not sure what installation method you specifically used, but GitHub Desktop can be installed using a tool like Chocolatey on Windows. What specific commands were you trying to run that did not work?
